src/Storage/Device/ArubaCloud.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Utopia\Storage\Device;
+
+use Utopia\Storage\Storage;
+
+final class ArubaCloud extends S3
+{
+    /**
+     * Region constants
+     *
```

Comment:
Would you please add a link to the reference material you used to determine the regions?